components/dropdown/Dropdown.js

```diff
@@ -44,16 +59,27 @@ const Dropdown = ({ isOpen, children, className, content, autoClose, autoCloseOu
         };
     }, []);
 
+    const alignClass = ALIGN_CLASSES[align];
+    const dropdownClassName = `dropDown ${alignClass ? alignClass : ''} pm-button`;
+    const contentClassName = `dropDown-content ${narrow ? 'dropDown-content--narrow' : ''}`;
+    const caretContent = caret && <Icon className="expand-caret" size={12} name="caret" />;
+
     return (
-        <div className="dropDown" ref={wrapperRef}>
-            <Button className={className} onClick={handleClick} aria-expanded={open}>
-                {content}
-            </Button>
-            {open ? (
-                <div className="dropDown-content" onClick={handleClickContent}>
-                    {children}
-                </div>
-            ) : null}
+        <div className={`${dropdownClassName} ${className}`} ref={wrapperRef}>
+            <button
+                title={title}
+                className="increase-surface-click"
+                aria-expanded={open}
+                onClick={handleClick}
+                type="button"
+            >
+                <span className="mauto">
+                    {content} {caretContent}
+                </span>
+            </button>
+            <div className={contentClassName} onClick={handleClickContent} hidden={!open}>
```

Comment:
Why keep the content of the `Dropdown` in the page? I suppose for animation?

Reply:
Yes, and we can attach an event handler for `onAnimationEnd` and remove it if we see that it becomes a performance issue to always render all of them. But I think it should be ok so I didn't add it for now